main.go

```diff
@@ -33,9 +33,8 @@ func main() {
 	filterPath := ""
 	flaggy.String(&filterPath, "f", "filter", "Path to filter on in `git log -- <path>`. When in filter mode, the commits, reflog, and stash are filtered based on the given path, and some operations are restricted")
 
-	dump := ""
-	flaggy.AddPositionalValue(&dump, "gitargs", 1, false, "Todo file")
-	flaggy.DefaultParser.PositionalFlags[0].Hidden = true
+	gitArgs := ""
+	flaggy.AddPositionalValue(&gitArgs, "gitargs", 1, false, "status/branch/log/stash is valid and each starts with Files/Branches/Commits/Stash panel.")
```

Comment:
This is tricky: we have a panel in the top left named status but by passing the argument as 'status' we're taking the user to the files panel instead. But I do like the idea of using 'status' given it's equivalent to typing 'git status'. Thoughts @HiromasaNojima @Ryooooooga @mark2185 ?

Perhaps we should rename the status panel to 'repo'

Reply:
@jesseduffield 
Users probably expect `lazygit status` behaves equivalent to `git status`. 
> Perhaps we should rename the status panel to 'repo'

So I agree with renaming.

Reply:
![Screen Shot 2022-05-21 at 19 40 35](https://user-images.githubusercontent.com/67505578/169648002-76ba2cd1-63e3-4c45-89cf-7b5a26354305.png)

I renamed only panel's title to 'Info' like this picture. If this is ok, I'll rename 'status' to 'info' in other places.